study-builder/studies/basil/i18n/en.conf

```diff
@@ -638,6 +638,8 @@
     "cancer_additional_primary_bone_cancer": "Primary bone cancer",
     "cancer_additional_rm_retinoblastoma": "Retinoblastoma (RB)",
     "cancer_additional_vhl_von_hippel_lindau": "Von hippel-lindau disease (VHL)",
+    "other_cancer": "[es] Other cancer",
+    "other_cancer_details": "[es] Other cancer details",
```

Comment:
Not a big deal because this option is invisible, but don't need [es] prefix.

Reply:
Removed [es] prefixes

Reply:
Please add all the es prefixes again to the es.conf file. They were not necessary in this specific location/file. We don't use them in the English file, only in the Spanish file to highlight text that we don't have a translation for. The requested change was just for these two lines.

Reply:
Re-fixed